api/v1/clustersecurityintentbinding_types.go

```diff
@@ -27,12 +27,19 @@ type ClusterSecurityIntentBindingSpec struct {
 
 // ClusterSecurityIntentBindingStatus defines the observed state of ClusterSecurityIntentBinding
 type ClusterSecurityIntentBindingStatus struct {
-	Status string `json:"status"`
+	Status               string `json:"status"`
+	NumberOfBoundIntents uint   `json:"numberOfBoundIntents"`
+	BoundIntents         string `json:"boundIntents"`
```

Comment:
Shouldn't BoundIntents be a slice ? You have a NumberofBoundIntents, so there are multiple BoundIntents..

Reply:
You're right, so do we want to show the actual intents in `kubectl get sib -o wide` or keep those in `kubectl describe sib` output?

Reply:
Done as suggested.